.envrc

```diff
@@ -0,0 +1,3 @@
+export GOOSE_DRIVER=sqlite3
+export GOOSE_DBSTRING=./go_sudoku.db
+export GOOSE_MIGRATION_DIR=./db/migrations
```

Comment:
Use direnv to manage `goose` env vars https://github.com/pressly/goose?tab=readme-ov-file#usage